src/Nethermind/Nethermind.Blockchain/Processing/ITransactionProcessingStrategy.cs

```diff
@@ -15,20 +15,17 @@
 //  along with the Nethermind. If not, see <http://www.gnu.org/licenses/>.
 // 
 
-using NUnit.Framework;
-using Nethermind.Core;
+using System;
 using System.Collections.Generic;
-using Nethermind.Mev.Data;
+using Nethermind.Core;
+using Nethermind.Core.Specs;
+using Nethermind.Evm.Tracing;
+using Nethermind.State;
 
-namespace Nethermind.Mev.Test
+namespace Nethermind.Blockchain.Processing
 {
-    [TestFixture]
-    public class MevBundleForRpcTests
+    public interface ITransactionProcessingStrategy
     {
-        [Test]
-        public void Can_create()
-        {
-            _ = new MevBundle(new List<Transaction>(), 0, 0, 0);
-        }
+        public TxReceipt[] ProcessTransactions(Block block, ProcessingOptions processingOptions, IBlockTracer blockTracer, BlockReceiptsTracer receiptsTracer, IReleaseSpec spec, EventHandler<TxProcessedEventArgs> TransactionProcessed);
```

Comment:
I be;live we don't need to pass blockTracer here and TransactionProcessed definitly is not a good thing